package.json

```diff
@@ -26,7 +26,7 @@
     "prepublish": "gulp transpile",
     "test": "npm run lint && ava",
     "lint": "xo",
-    "format": "prettier --single-quote --no-semi --write *.js {src,test}/*.js",
+    "format": "prettier --single-quote --no-semi --write *.js {src,test}/*.js test/fixtures/*-prettier.js",
```

Comment:
the thing is that @leo found this bug because of prettier but the formatting which caused the issue can also be done by hand